test/common/tracing/http_tracer_impl_test.cc

```diff
@@ -475,6 +475,28 @@ TEST(HttpTracerImplTest, BasicFunctionalityNodeSet) {
   tracer.startSpan(config, request_headers, request_info);
 }
 
+TEST(HttpTracerImplTest, TestOperationName) {
+  LocalInfo::MockLocalInfo local_info;
+  SystemTime time;
+  Http::AccessLog::MockRequestInfo request_info;
+  EXPECT_CALL(request_info, startTime()).WillOnce(Return(time));
```

Comment:
This should be part of the default mock via ON_CALL, can you add this to default mock and clean this up elsewhere in this file.